cpp/meson.options

```diff
@@ -36,11 +36,18 @@ option(
     value: false,
 )
 
+option(
+    'integration',
+    type: 'boolean',
+    description: 'Build the Arrow integration test executables',
+    value: false,
+)
+
 option(
     'ipc',
     type: 'boolean',
     description: 'Build the Arrow IPC extensions',
-    value: false,
+    value: true,
```

Comment:
Why do we need this?

Reply:
Rebase mistake - thanks!

Reply:
Errr...actually this is intentional. It looks like `ARROW_IPC` is `ON` by default in the CMake configuration. I can split this off into a separate PR if needed though

Reply:
Ah, OK. Could you split this off into a separated PR for easy to track later?